x/evm/keeper/grpc_query.go

```diff
@@ -583,3 +583,20 @@ func (k *Keeper) traceTx(
 
 	return &result, txConfig.LogIndex + uint(len(res.Logs)), nil
 }
+
+// BaseFee implements the Query/BaseFee gRPC method
+func (k Keeper) BaseFee(c context.Context, _ *types.QueryBaseFeeRequest) (*types.QueryBaseFeeResponse, error) {
```

Comment:
can you create tests for the base fee query?

Reply:
done